mflowgen/Tile_PE/custom-init/outputs/floorplan.tcl

```diff
@@ -11,23 +11,16 @@
 #-------------------------------------------------------------------------
 
 # Density target: width will be adjusted to meet this cell density
-set core_density_target 0.70; # Placement density of 70% is reasonable
+set core_density_target 0.65; # Placement density of 70% is reasonable
```

Comment:
Should we change the comment to match the code?

Reply:
done